rskj-core/src/test/java/co/rsk/peg/ReleaseTransactionBuilderTest.java

```diff
@@ -61,6 +60,7 @@ public void createBuilder() {
             changeAddress,
             Coin.MILLICOIN.multiply(2),
             activations);
+        btcContext = new Context(NetworkParameters.fromID(NetworkParameters.ID_REGTEST));
```

Comment:
I wonder if we should maybe allow an externally defined btcContext. This could be useful to verify Testnet/Mainnet specific conditions